tests/AuxTest.ts

```diff
@@ -473,7 +564,7 @@ export const getParsedDidDocument = (didKey: DidKey): DIDDocument => {
 };
 
 export const getPublicJWKFromDid = async (did: string): Promise<JWK> => {
-  const API_BASE_URL = process.env.WALLET_API_URL || "https://api.vidchain.net";
+  const API_BASE_URL = process.env.WALLET_API_URL || "https://dev.vidchain.net";
```

Comment:
I would leave the default endpoint to be `api` and if you want to use another one, just set it on your `.env`